tests/test_protocols/test_generator.py

```diff
@@ -468,19 +467,19 @@ def setUp(self):
         protocol_specification = mock.Mock()
         protocol_specification.name = "name"
 
-    @mock.patch(
-        "aea.protocols.generator.common._get_sub_types_of_compositional_types",
-        return_value=["some"],
-    )
-    def test__includes_custom_type_positive(self, *mocks):
-        """Test _includes_custom_type method positive result."""
-        content_type = "pt:union[pt:str]"
-        result = not _is_composition_type_with_custom_type(content_type)
-        self.assertTrue(result)
-
-        content_type = "pt:optional[pt:str]"
-        result = not _is_composition_type_with_custom_type(content_type)
-        self.assertTrue(result)
+    # @mock.patch(
```

Comment:
Why removed?

Reply:
This one tested `_is_composition_type_with_custom_type` which was removed in favour of the more comprehensive validation function.